.github/workflows/ci.yml

```diff
@@ -18,12 +18,7 @@ jobs:
       - run: cd backend && poetry install
       - name: Add i386 architecture and install dependencies
         run: |-
-          sudo dpkg --add-architecture i386
-          sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 86B72ED9
-          sudo add-apt-repository 'deb [arch=amd64] https://mirror.mxe.cc/repos/apt focal main'
           sudo apt -qq update
-          sudo apt install -y --allow-downgrades libpcre2-8-0=10.34-7
-          sudo apt install -y wine
```

Comment:
I reckon you either need `i386` arch + `libstdc++6:i386` OR, per the Wibo build, `g++-multilib` -- else you wont be able to run the 32bit elf